src/renderer/lib/telemetry.js

```diff
@@ -118,29 +119,60 @@ function getApproxNumTorrents (state) {
 }
 
 // An uncaught error happened in the main process or in one of the windows
-function logUncaughtError (procName, err) {
+function logUncaughtError (procName, e) {
   // Not initialized yet? Ignore.
   // Hopefully uncaught errors immediately on startup are fixed in dev
   if (!telemetry) return
 
-  var message, stack
-  if (err instanceof Error) {
-    message = err.message
-    // Remove the first part of each file path in the stack trace.
-    // - Privacy: remove personal info like C:\Users\<full name>
-    // - Aggregation: this lets us find which stacktraces occur often
-    stack = err.stack.replace(/\(.*app.asar/g, '(...')
+  var message
+  var stack = ''
+  if (e.message) {
```

Comment:
I don't think that `e` can be null/undefined now, but maybe we should check that anyway to be sure. We can always remove it if we don't see anything in the telemetry reports, but if it happens the report will never be logged or sent.